src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/EditorTests.java

```diff
@@ -0,0 +1,167 @@
+
+  @Test(groups = "MercuryCuratedEditorTests_001")
+  @Execute(onWikia = "mercuryemptycceditor")
+  public void MercuryCuratedEditorTest_001_addAndSaveItemToFeaturedContent() {
+    CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
+    EditorHomePageObject editorHomePageObject = new EditorHomePageObject(driver);
+
+    Boolean result = !curatedMainPagePageObject.isFeaturedContentVisible();
+    PageObjectLogging.log(
+        "Featured Content",
+        MercuryMessages.INVISIBLE_MSG,
+        MercuryMessages.VISIBLE_MSG,
+        result
+    );
+
+    curatedMainPagePageObject.navigateToUrlWithPath(wikiURL, MAIN_EDIT_ROOT);
+    ItemFormPageObject itemFormPageObject = editorHomePageObject.clickAddFeaturedContent();
+    itemFormPageObject.typeDisplayName(ITEM_DISPLAY_NAME);
+    itemFormPageObject.typePageName(ITEM_PAGE_NAME);
+
+    UploadImageModalComponentObject upload = itemFormPageObject.clickOnImage();
+    SearchForImagePageObject search = upload.clickSearchForImageButton();
+    search.type(ON_WIKI_IMAGE_PREFIX);
+    CroppingToolPageObject croppingTool = search.clickOnImage(0);
+    croppingTool.clickDone();
+
+    itemFormPageObject.waitForDeleteButtonToBeVisible();
+    itemFormPageObject.clickDone();
+    editorHomePageObject.waitForAddCategoryButtonToBeVisible();
+    editorHomePageObject.publish();
+
+    result = curatedMainPagePageObject.isFeaturedContentVisible();
+    PageObjectLogging.log(
+        "Featured Content",
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+  }
+
+  @Test(groups = "MercuryCuratedEditorTests_002")
+  @Execute(onWikia = "mercuryemptycceditor")
+  public void MercuryCuratedEditorTest_002_addAndSaveSection() {
+    CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
+    curatedMainPagePageObject.isCuratedContentVisible();
+    new BasePageObject(driver).navigateToUrlWithPath(wikiURL, MAIN_EDIT_ROOT);
+    EditorHomePageObject home = new EditorHomePageObject(driver);
+    SectionFormPageObject section = home.clickAddSection();
+    section.typeDisplayName(SECTION_NAME);
+    UploadImageModalComponentObject upload = section.clickOnImage();
+    SearchForImagePageObject search = upload.clickSearchForImageButton();
+    search.type(ON_WIKI_IMAGE_PREFIX);
+    CroppingToolPageObject croppingTool = search.clickOnImage(0);
+    croppingTool.clickDone();
+    SectionItemListPageObject sectionItems = section.clickDone();
+
+    CategoryFormPageObject category = sectionItems.clickAddCategory();
+    category.typeDisplayName(CATEGORY_DISPLAY_NAME);
+    category.typeCategoryName(CATEGORY_NAME);
+    upload = category.clickOnImage();
+    search = upload.clickSearchForImageButton();
+    search.type(ON_WIKI_IMAGE_PREFIX);
+    croppingTool = search.clickOnImage(0);
+    //category = (CategoryFormPageObject) croppingTool.clickDone();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)